__tests__/api.test.js

```diff
@@ -90,3 +90,25 @@ describe("2. GET /api/reviews/:review_id", () => {
       });
   });
 });
+
+describe("GET api/users", () => {
+  it("should return status: 200, and an array of catagory objects containing the correct keys", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(response => {
+        const users = response.body.users;
+        expect(users).toBeInstanceOf(Array);
```

Comment:
we don't technically need this as line 103 implies that it is an array (forEach only works on arrays)